src/Item/Item.model.ts

```diff
@@ -13,6 +14,17 @@ export class Item extends Model<ItemAttributes> {
         WHERE id = ANY(${ids})`)
   }
 
+  static findDBApprovalDataByCollectionId(collectionId: string) {
+    return this.query<DBItemApprovalData>(SQL`
+      SELECT items.id, items.urn_suffix, item_curations.local_content_hash
+        FROM ${raw(this.tableName)} items
+        JOIN ${raw(
+          ItemCuration.tableName
+        )} item_curations ON items.id = item_curations.item_id AND
+        WHERE items.collection_id = ${collectionId}
+          AND item_curations.status = ${CurationStatus.PENDING}`)
```

Comment:
I think we need a DB restriction where an item can't have more than one curation in the pending status. Would you mind creating a ticket for this?

Reply:
Done! https://github.com/decentraland/builder-server/issues/432